src/market.py

```diff
@@ -183,14 +205,66 @@ def parse_fill_event(self, event) -> FilledOrder:
             fee_cost=event.native_fee_or_rebate * (1 if event.event_flags.maker else -1),
         )
 
-    def place_order(self, order_params: NewOrderParams):
-        pass
+    def place_order(self, order_params: PlaceOrderParams):
+        txs, signers = self.make_place_order_transaction(order_params)
+        return self._send_transaction(txs, *signers)
 
-    def make_place_order_transaction(self, order_params: NewOrderParams) -> Tuple[Transaction, List[PublicKey]]:
-        pass
+    def make_place_order_transaction(self, order_params: PlaceOrderParams) -> Tuple[Transaction, List[Account]]:
```

Comment:
Can we consolidate this to `place_order`? What's the reason for them to be separate? We already have `make_place_order_instruction`.

Reply:
good point. fixed.